bakery/src/scripts/Dockerfile

```diff
@@ -28,6 +28,12 @@ RUN wget https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pando
     dpkg -i /tmp/pandoc.deb && \
     rm -f /tmp/pandoc.deb
 
-COPY ./*.py ./*.json /code/scripts/
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN apt-get install -y nodejs
+
+COPY ./*.py ./*.js ./*.json /code/scripts/
 COPY ./gdoc/ /code/gdoc/
 RUN pip install /code/scripts/.
+
+RUN npm --prefix /code/scripts install /code/scripts
+RUN npm install forever@3.0.2 -g
```

Comment:
Minor: Would it make developer iterations faster (e.g. when doing `docker build src/scripts/.` after a code change) if we move this to just after line 32 🤷‍♂️ ?

Reply:
I changed the order of commands in the last commit. But as you said I thing it has maybe only minor impact.